### ISSUE TITLE
Rule Proposal: No nested isClient, isServer, isCordova blocks

### ISSUE THREAD

**Issue by dferber90** [dferber90/eslint-plugin-meteor#26]
The following nesting style makes no sense because the code will never run anywhere. Prevent it.

``` js
if (Meteor.isClient) {
  if (Meteor.isServer) {
    // code in this block will never run
  }
}
```

This goes for all combinations of `isClient`, `isServer` and `isCordova`. The rule should warn on the inner `ifStatement`.

Potential rule name: `no-nested-env-checks`?
